docs/docs/index.md

```diff
@@ -1,36 +1,24 @@
-# Welcome
+# **Welcome**
 
-Hydra is a layer-two scalability solution for Cardano, which aims to increase
-the speed of transactions (low latency, high throughput) and minimize
-transaction cost.
+This section is a user manual for Hydra node operators. 
```

Comment:
This suggests that hydra node operators are a definite group of people. We do not, or have not used that stance in the past. Notably, there is not one Hydra network and hence this is different to the stake pool operators on the Cardano network for example.

Reply:
How about: 'This section is a manual for users interested in running a Hydra node'